ckanext/switzerland/plugins.py

```diff
@@ -263,7 +263,7 @@ def edit(self, grp_dict):
         :param grp_dict:
         :return:
         """
-        ogdch_backend_helpers.ogdch_add_users_to_groups(None, None)
+        ogdch_logic.ogdch_add_users_to_groups(None, {})
```

Comment:
Why would that be necessary at all. Can't we just remove this. When a group is edited I don't see why that should be done. It might make sense when creating a group but not when editing it. WDYT?

Reply:
This method is called when creating a new group as well as when editing an old one, which is how I discovered the bug.